SQL/New_patches/2021-09-13_fix_project_primary_key.sql

```diff
@@ -0,0 +1,3 @@
+-- Add a unique constraint on Project.Name
+CREATE UNIQUE INDEX u_ProjectID ON `Project` (`ProjectID`);
```

Comment:
Should this be Name instead of ProjectID?

Reply:
You are absolutely right :)

Reply:
```suggestion
CREATE UNIQUE INDEX `u_ProjectName` ON `Project` (`Name`);
```